openmmtools/integrators.py

```diff
@@ -1856,19 +1856,19 @@ def __init__(self, *args, **kwargs):
 
     def reset(self):
         super(ExternalPerturbationLangevinIntegrator, self).reset()
-        self.setGlobalVariableByName('first_step', 0)
+        self.setGlobalVariableByName('step', 0)
 
     def _add_global_variables(self):
         super(ExternalPerturbationLangevinIntegrator, self)._add_global_variables()
         self.addGlobalVariable("perturbed_pe", 0)
         self.addGlobalVariable("unperturbed_pe", 0)
-        self.addGlobalVariable("first_step", 0)
+        self.addGlobalVariable("step", 0)
 
     def _add_integrator_steps(self):
         self.addComputeGlobal("perturbed_pe", "energy")
         # Assumes no perturbation is done before doing the initial MD step.
-        self.beginIfBlock("first_step < 1")
-        self.addComputeGlobal("first_step", "1")
+        self.beginIfBlock("step < 1")
+        self.addComputeGlobal("step", "1")
```

Comment:
To match the other integrators, why not increment `step` each step?

Reply:
ok